addons/point_of_sale/models/pos_config.py

```diff
@@ -224,12 +224,12 @@ def _compute_current_session_user(self):
     @api.constrains('company_id', 'stock_location_id')
     def _check_company_location(self):
         if self.stock_location_id.company_id and self.stock_location_id.company_id.id != self.company_id.id:
-            raise ValidationError(_("The company of the stock location is different than the one of point of sale"))
+            raise ValidationError(_("The stock location and the point of sale must belong to the same company"))
 
     @api.constrains('company_id', 'journal_id')
     def _check_company_journal(self):
         if self.journal_id and self.journal_id.company_id.id != self.company_id.id:
-            raise ValidationError(_("The company of the sales journal is different than the one of point of sale"))
+            raise ValidationError(_("The sales journal and the point of sale must belong to the same company"))
```

Comment:
Once again, a . is missing : "The sales journal and the point of sale must belong to the same company."